wagtail/research/comparisons-to-existing-tools.md

```diff
@@ -0,0 +1,19 @@
+# Comparisons to existing tools
+
+Over the past year DataMade has employed two CMS platforms:
+
+- [Django CMS](https://www.django-cms.org/en/) for [Neighborhood Opportunity Fund](https://github.com/datamade/neighborhood-fund)
+- [Wagtail](https://wagtail.io/) for [LISC CDNA](https://github.com/datamade/lisc-cnda) and the [Lugar Center Oversight Index](https://github.com/datamade/committee-oversight)
+
+
+## Wagtail vs. Django CMS
+
+Both Wagtail and Django CMS are built for use with Django, but we were surprised to learn that they ask developers to employ substantially different abstractions—the way the two CMSs structure their data across views and models is quite different.
```

Comment:
A description of the data structure and editing experience for each CMS would better support your points below.